src/plugins/condition/components/Criterion.vue

```diff
@@ -77,8 +89,10 @@ export default {
     data() {
         return {
             telemetryMetadata: {},
+            telemetryMetadataOptions: {},
             operations: OPERATIONS,
-            isInputOperation: false,
+            filteredOps: [],
```

Comment:
this looks like a good candidate for a computed property

Reply:
done